flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/sink/ValuesDataSink.java

```diff
@@ -66,7 +62,7 @@ public MetadataApplier getMetadataApplier() {
     }
 
     /** an e2e {@link Sink} implementation that print all {@link DataChangeEvent} out. */
-    private static class ValuesSink implements StatefulSink<Event, TableSchemaState> {
```

Comment:
TableSchemaState could remove now ?

Reply:
Yeah, we don't need to add Schema message to State as CreateTableEvent will be sent firstly.